tests/certs_test.go

```diff
@@ -338,64 +383,6 @@ var _ = Describe("Certs", func() {
 				Entry("a non-wildcard cert to a non-wildcard domain",
 					domains["bar"], certs["bar"].Name, certs["bar"].CertPath, certs["bar"].KeyPath),
 			)
-
-			DescribeTable("2 certs (1 wildcard), 3 domains (1 wildcard)",
```

Comment:
removed this (was not a fan of the convoluted logic anyhow) but scenario covered in the new one added, with the benefit of adding a test around https://github.com/deis/workflow/pull/492 / https://github.com/deis/workflow/pull/499